apis/enigma-mocker/src/from-generic-objects.js

```diff
@@ -16,7 +16,7 @@ export default function fromGenericObjects(genericObjects, options = {}) {
     destroySessionObject: async () => {},
     getObject,
     getAppLayout,
-    getListObject: async () => {},
+    getListObject: async (props) => getObject(props.qInfo?.qId),
```

Comment:
I wonder why I added this in the first place, as it is only part of the client mixins. Think it was something with embed, but unsure. This is fine though.